runtime/opt/taupage/init.d/04-configure-network-interfaces.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+
+import boto.ec2
+import boto.utils
+import time
+import sys
+import subprocess
+import netifaces
+from netaddr import IPAddress
+
+from taupage import configure_logging, get_config
+
+
+def instance_id():
```

Comment:
the returned values won't change and thus can be cached statically to avoid repeated lookups / AWS API calls.

Reply:
Copied from https://github.com/zalando-stups/taupage/blob/master/runtime/opt/taupage/init.d/10-prepare-disks.py#L18 but will adjust

Reply:
Then it is fine.